learn/core_concepts/relevancy.md

```diff
@@ -1,44 +1,48 @@
 
-The order in which they are applied has a significant impact on the search results. The first rules being the most impactful and the last one the least. The default order has been chosen because it meet most standard needs. **This order can be changed in the settings**.
+**The order in which ranking rules are applied matters.** The first rule in the array has the most impact, and the last rule has the least. Our default configuration has been chosen because it meets most standard needs. [This order can be changed in the settings](/reference/api/ranking_rules.md#update-ranking-rules).
 
-By default, ranking rules are executed in the following order:
+### Built-in rules
+
+MeiliSearch contains six built-in ranking rules: **typo, words, proximity, attribute, words position, and exactness**, in that default order.
 
 **1. Typo**
-Results are sorted by **increasing number of typos**: find documents that match query terms with fewer typos first.
+Results are sorted by **increasing number of typos**. Returns documents that match query terms with fewer typos first.
 
 **2. Words**
-Results are sorted by **decreasing number of matched query terms** in each matching document: find documents that contain more occurrences of the query terms first.
+Results are sorted by **decreasing number of matched query terms**. Returns documents that contain all query terms first.
 
 ::: warning
 
-It is now mandatory that all query terms are present in the returned documents. This rule does not impact search results yet. <Badge text="soon" type="warn"/>
+For now, it is mandatory that all query terms are present in returned documents. Therefore, this rule does not impact search results yet. <Badge text="soon" type="warn"/>
 
 :::
 
 **3. Proximity**
-Results are sorted by **increasing distance between matched query terms**: find documents that contain more query terms found close together (close proximity between two query terms) and appearing in the original order specified in the query string first.
+Results are sorted by **increasing distance between matched query terms**. Returns documents where query terms occur close together and in the same order as the query string first.
 
 **4. Attribute**
-Results are sorted according to the **[attribute ranking order](/learn/core_concepts/relevancy.md#attribute-ranking-order)**: find documents that contain query terms in more important attributes first.
+Results are sorted according to the **[attribute ranking order](/learn/core_concepts/relevancy.md#attribute-ranking-order)**. Returns documents that contain query terms in more important attributes first.
 
 **5. Words position**
-Results are sorted by **the position of the query words in the attributes**: find documents that contain query terms earlier in their attributes first.
+Results are sorted by **the location of the query word in the field**. Returns documents that contain query terms close to the beginning of the field first.
```

Comment:
More of a question than an actual suggestion: wouldn't `term` be more appropriate than `word`?

Reply:
Hmm, I'm not sure. Do you mean:

```suggestion
Results are sorted by **the location of the query term in the field**. Returns documents that contain query terms close to the beginning of the field first.
```

Or are you talking about the name of the ranking rule itself?

Reply:
Both, actually. I was wondering whether there was any reason we used `word` in this section whilst preferring `term` in most other places.

Reply:
It's a good point. "Word" might be a bit more familiar to non-tech people, but it's not as accurate as "term".